CONTRIBUTING.md

```diff
@@ -13,6 +13,8 @@ You can start contributing to any of the following library groups from GitHub:
   - [Room](https://developer.android.com/topic/libraries/architecture/room)
   - [WorkManager](https://developer.android.com/topic/libraries/architecture/workmanager)
 
+For the list of bugs that you can contribute towards, please take a look at the official [bug bounty list](http://goo.gle/androidx-bug-bounty).
```

Comment:
```suggestion
Not sure where to start? Take a look at the official [bug bounty list](http://goo.gle/androidx-bug-bounty)!
```

Reply:
Maybe something like this so that it doesn't seem like we won't accept contributions outside things we've already created bugs for?

Reply:
Done.

Reply:
maybe call this bug/feature bounty list because most of them are actually features ?

Reply:
Done.